src/app/config/config.c

```diff
@@ -6405,60 +6407,59 @@ port_parse_config(smartlist_t *out,
         }
 
         if (no) {
-          isolation &= ~isoflag;
+          cfg_def->entry_cfg.isolation_flags &= ~isoflag;
         } else {
-          isolation |= isoflag;
+          cfg_def->entry_cfg.isolation_flags |= isoflag;
         }
       } SMARTLIST_FOREACH_END(elt);
     }
 
-    if (port)
-      got_nonzero_port = 1;
-    else
-      got_zero_port = 1;
+    if (port) got_nonzero_port = 1;
+    else got_zero_port = 1;
```

Comment:
You don't have to change anything in response to this comment.

But here are a few coding style hints:
1. Try to match the surrounding style
2. We try to keep behaviour changes and style changes separate. If you want to change the C style to make it more consistent, make that change in a separate commit.

We're also working on a better C style guide, and automated C styling.